src/main/java/uk/gov/ons/ctp/common/event/EventPublisher.java

```diff
@@ -319,10 +319,10 @@ private void sendToRabbit(
     EventType eventType = genericEvent.getEvent().getType();
     log.with("eventType", eventType)
         .with("routingKey", routingKey)
-        .debug("Sending message {}", loggingMsgSuffix);
+        .info("Sending message to rabbit {}", loggingMsgSuffix);
     sender.sendEvent(routingKey, genericEvent);
     log.with("eventType", eventType)
         .with("routingKey", routingKey)
-        .debug("Message sent successfully");
+        .info("Message sent successfully by rabbit");
```

Comment:
not sure we need the bit about rabbit, but if we have it the english would be better if it said "to rabbit" rather than "by rabbit" .

Reply:
will change it "to rabbit" thought it is useful to add it as "Message sent successfully" becomes too generic.

Reply:
changes done